opentelemetry/src/test/scala/zio/telemetry/opentelemetry/TracingTest.scala

```diff
@@ -155,16 +155,18 @@ object TracingTest extends DefaultRunnableSpec {
           val log =
             for {
               _ <- UIO.unit.addEvent("message")
+              fiber <- ZIO
+                        .sleep(duration)
```

Comment:
This sleep was a workaround. it should be possible to simplify `log` to 
```scala
TestClock.adjust(duration)
  .addEvent("message")
  .addEventWithAttributes(
    "message2",
    Map(
      "msg"  -> AttributeValue.stringAttributeValue("message"),
      "size" -> AttributeValue.longAttributeValue(1)
    )
  )
```

The comment above also is resolved then.

Reply:
Hmm if I do that, the test fails because we get `(0,message,Map())` for the first message, instead of `(1000000,message,Map())`, which makes sense. Should I change the expectation?

Reply:
Otherwise I change it to:
```scala
ZIO.unit.addEvent("message") *>
  TestClock
    .adjust(duration)
    .addEventWithAttributes(
      "message2",
      Map(
        "msg"  -> AttributeValue.stringAttributeValue("message"),
        "size" -> AttributeValue.longAttributeValue(1)
      )
    )
```
Which one is better?

Reply:
Ah, right. I think the one you propose is better.

Reply:
Okay, I fixed both.